src/err/mod.rs

```diff
@@ -198,7 +198,9 @@ impl PyErr {
     ///
     /// Python::with_gil(|py| {
     ///     // Case #1: Exception object
-    ///     let err = PyErr::from_value(PyTypeError::new_err("some type error").value(py));
+    ///     let err = PyErr::from_value(PyTypeError::new_err("some type error")
+    ///         .value_bound(py)
+    ///         .into_gil_ref());
```

Comment:
I think to remove this `into_gil_ref()` we would need a `PyErr::from_value_bound` method.